### DISCUSSION TITLE
New Feature: support for color in tabLabelStyle

### DISCUSSION THREAD

**Discussion by dlindstrm** [callstack/react-native-bottom-tabs#212] [Ideas]
### Why it is needed?

It would be nice to be able to have a different color of the label on Android. For example green icon and black label, like Google Maps:
<img width="1085" alt="Skärmavbild 2024-12-13 kl  10 17 22" src="https://github.com/user-attachments/assets/9224232a-fc38-43a5-8d17-0bdd7db0951f" />

**Comment by okwasniewski** [comment]
Hey, thanks for opening the feature request. I'm moving this to discussions because all feature requests will now be under the "Ideas 💡" tab in GitHub discussions.
